vite.config.js

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'vite'
+
+export default defineConfig({
```

Comment:
Can you please lint this file with prettier

Reply:
@wolfy1339 
done